base/_fonts.scss

```diff
@@ -0,0 +1,39 @@
+// -----------------------------------------------------------------------------
```

Comment:
I like the sass docs format for file annotations. We should follow it for all of our files:
http://sassdoc.com/file-level-annotations/